draft-ietf-ccamp-rfc9093-bis.txt

```diff
@@ -277,77 +277,251 @@ Internet-Draft           Yang for Layer 0 Types               March 2022
+        The key words 'MUST', 'MUST NOT', 'REQUIRED', 'SHALL', 'SHALL
 
 
 
-Zheng, et al.           Expires 17 September 2022               [Page 6]
+Zheng, et al.            Expires 11 January 2023                [Page 6]
 
-Internet-Draft           Yang for Layer 0 Types               March 2022
+Internet-Draft           Yang for Layer 0 Types                July 2022
+
+
+        NOT', 'SHOULD', 'SHOULD NOT', 'RECOMMENDED', 'NOT RECOMMENDED',
+        'MAY', and 'OPTIONAL' in this document are to be interpreted as
+        described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
+        they appear in all capitals, as shown here.";
 
+   // RFC Ed.: replace XXXX with actual RFC number and remove
+   // this note
 
+   // replace the revision date with the module publication date
+   // the format is (year-month-day)
+     revision 2022-07-10 {
+       description
+         "To be updated";
+       reference
+         "RFC XXXX: A YANG Data Model for Layer 0 Types";
+     }
+     revision 2021-08-13 {
+       description
+         "Initial version";
+       reference
+         "RFC 9093: A YANG Data Model for Layer 0 Types";
      }
 
    /*
     * Identities
     */
 
+     identity l0-grid-type {
+       description
+         "Layer 0 grid type";
+       reference
+         "RFC 6163: Framework for GMPLS and Path Computation Element
+          (PCE) Control of Wavelength Switched Optical Networks (WSONs),
+          ITU-T G.694.1 (10/2020): Spectral grids for WDM applications:
+          DWDM frequency grid,
+          ITU-T G.694.2 (12/2003): Spectral grids for WDM applications:
+          CWDM wavelength grid";
+     }
+
+     identity flexi-grid-dwdm {
+       base l0-grid-type;
+       description
+         "Flexi-grid";
+       reference
+         "RFC 7698: Framework and Requirements for GMPLS-Based Control
+          of Flexi-Grid Dense Wavelength Division Multiplexing (DWDM)
+          Networks,
+          ITU-T G.694.1 (10/2020): Spectral grids for WDM applications:
+
+
+
+Zheng, et al.            Expires 11 January 2023                [Page 7]
+
+Internet-Draft           Yang for Layer 0 Types                July 2022
+
+
+          DWDM frequency grid";
+     }
+
+     identity wson-grid-dwdm {
+       base l0-grid-type;
+       description
+         "DWDM grid";
+       reference
+         "RFC 6163:Framework for GMPLS and Path Computation Element
+          (PCE) Control of Wavelength Switched Optical Networks (WSONs),
+          ITU-T G.694.1 (10/2020): Spectral grids for WDM applications:
+          DWDM frequency grid";
+     }
+
+     identity wson-grid-cwdm {
+       base l0-grid-type;
+       description
+         "CWDM grid";
+       reference
+         "RFC 6205: Generalized Labels for Lambda-Switch-Capable (LSC)
+          Label Switching Routers,
+          ITU-T G.694.2 (12/2003): Spectral grids for WDM applications:
+          CWDM wavelength grid";
+     }
+
+     identity dwdm-ch-spc-type {
```

Comment:
It is not a blocking point fro this version, but the name is not friendly . Please turn into e.g. dwdm-ch-spacing-type.
It is longer but clear .